src/actions/actions.js

```diff
@@ -195,31 +184,16 @@ export function updateInputContent(text) {
   }
 }
 
-
-export function changeCellType(cellType, language = 'js') {
-  return (dispatch, getState) => {
-    if ((getSelectedCell(getState()).cellType !== cellType
-      || getSelectedCell(getState()).language !== language)) {
-      dispatch({
-        type: 'CHANGE_CELL_TYPE',
-        cellType,
-        language,
-      })
-    }
-  }
-}
-
 export function addLanguage(languageDefinition) {
   return (dispatch) => {
     const {
       keybinding,
-      languageId,
       codeMirrorMode,
     } = languageDefinition
     if (keybinding.length === 1 && (typeof keybinding === 'string')) {
       addLanguageKeybinding(
         [keybinding],
-        () => dispatch(changeCellType('code', languageId)),
+        () => {},
```

Comment:
language key bindings are not needed at all in direct JSMD mode. Can we cut this whole containing block (so that `addLangaugeKeybinding` is never even called) rather than just putting in the dummy function?